modules/components/src/Arranger/Aggregations.js

```diff
@@ -8,7 +8,7 @@ const BaseWrapper = ({ className, ...props }) => (
 );
 
 const Aggregations = ({
-  onTermSelected = () => {},
+  onTermSelected,
```

Comment:
this prop can just be called `onValueChange`, the `onValueChange` passed into `aggComponents` isn't differentiating between types.